docs/mkdocs.yml

```diff
@@ -92,298 +91,229 @@ plugins:
                 - "!^_"
 
 nav:
-  - LangGraph:
-    - index.md
+  - Guides:
+    - Home: index.md
     - Get started:
-      - Learn the basics: tutorials/introduction.ipynb
+      # - TO DO: Quickstart with Prebuilt Agents
+      - LangGraph basics: tutorials/introduction.ipynb
+      - Key features: concepts/high_level.md
+      - Glossary: concepts/low_level.md
```

Comment:
might need a better name for this, i know @eyurtsev doesn't like this one

Reply:
It might make more sense to repurpose this one as "Key features" and then combine the "Why LangGraph" content onto the home/index page. Thoughts?